lib/spack/spack/test/spec_yaml.py

```diff
@@ -8,7 +8,10 @@
 The YAML and JSON formats preserve DAG information in the spec.
 
 """
+from __future__ import print_function
```

Comment:
I'm not seeing any `print()` calls in this file; is this still necessary?

Reply:
It isn’t but it doesn’t hurt :)